### PR TITLE
examples: fix broken gui helpfiles

### LINKED ISSUE #560: Most examples in 'examples/GUI examples' do not work
Nearly none of the examples in 'examples/GUI examples' work out of the box. Some work with trivial modifications. Some invoke objects that are not implemented in GUI.qt

These should be removed, or fixed if they are going to be left in - they're the first thing a new user will try, they should work out of the box.

### PR DESCRIPTION
**Pull request by telephon**
This fixes #560.

### CODE REVIEW

**Comment by brianlheim** [comment]
Thanks! Will test later today.

**Comment by brianlheim** [comment]
Okay first of all, this is just super fun. I forgot how easy it is to get excited about GUI tricks!

Second of all, I'm not sure what the examples in `strike.scd` are supposed to be doing. At the very least the "Tetris" example does not seem to be acting like a normal tetris game, and I'm not getting any sound out of it.

Third, would you consider making the examples in `x-y-plot.scd` play with sound? That was something I immediately did myself out of curiosity by adding `.play` before `.scope`. I get that it's a GUI example so not a big deal.

**Comment by telephon** [comment]
– I've simplified the sound example so you have to run only one extra bit of code.
– the x-y-plots make some sound now.

**Comment by telephon** [comment]
>Second of all, I'm not sure what the examples in strike.scd are supposed to be doing. At the very least the "Tetris" example does not seem to be acting like a normal tetris game

This is poetic, it is a portrait of tetris. Sorry that your expectations were disappointed! :)

File: examples/GUI examples/strike.scd [hunk 1]
```diff
@@ -150,8 +143,7 @@ q.sendData = {|q, data|
 		};
 
 		LPF.ar(ugens.mean, 3000) * 0.2 ! 2
-	}
+	}).play;
 
 };
-);
-);
+);
```
Comment (brianlheim) [inline examples/GUI examples/strike.scd]: missing a newline here!

Comment (brianlheim) [inline examples/GUI examples/strike.scd]: thanks @telephon :)

**Comment by brianlheim** [comment]
> This is poetic, it is a portrait of tetris. Sorry that your expectations were disappointed! :)

Hmm, could you describe what it is supposed be doing? What I see when I run it is a single squiggle like from the first example descend about 2/3 of the way down the screen, then disappear, then a few seconds later there are a few momentary vertical lines, then the cycle repeats. When I run the "with sound" code, I don't hear anything and no synthdefs seem to be generated.

**Comment by telephon** [comment]
… sorry, try again if you like.

**Comment by brianlheim** [comment]
Ahhhh thank you! I definitely needed the "click and drag" instruction, hahaha. I had no idea these were interactive, although I probably should have guessed from seeing ".mouseUpAction_".

**Pull request merged**